test/jdk/sun/security/provider/PolicyParser/ExtDirsDefaultPolicy.java

```diff
@@ -26,25 +26,34 @@
  * @bug 4993819
  * @summary standard extensions path is hard-coded in default
  *      system policy file
- * @run main/manual ExtDirsDefaultPolicy
+ * @run main ExtDirsDefaultPolicy
  */
 
 /*
- * Run this test manually with:
- * java    -Djava.security.manager \
- *         -Djava.ext.dirs=./ExtDirsA:./ExtDirsB \
- *         -Djava.security.debug=parser \
- *         ExtDirsDefaultPolicy
+ * @test
+ * @bug 4993819
+ * @summary standard extensions path is hard-coded in default
+ *      system policy file
  *
- * To test other varients of the ${{java.ext.dirs}} protocol, remove
- * the grant statement for java.ext.dirs in $JAVA_HOME/lib/security/java.policy
- * and then run against the 3 different policy files.
+ * @run main/othervm/policy=ExtDirs1.policy/secure=default ExtDirsDefaultPolicy
+ */
+
+/*
+ * @test
+ * @bug 4993819
+ * @summary standard extensions path is hard-coded in default
+ *      system policy file
+ *
```

Comment:
No need to add again these statement again.

Reply:
Are you saying that we don't need to run the ExtDir2.policy and ExtDir3.policy?

Reply:
Its fine, ignore my previous comment.

Reply:
you can add multiple run tags under same test tag, no need to repeat tag, bug, and summary tags.